docker/Dockerfile

```diff
@@ -51,24 +57,17 @@ RUN	go mod download
 
 COPY	.	.
 
-ARG	BUILD_TAGS
```

Comment:
i remember why i had it put so low in `Dockerfile`. if we move this to start of the file, each run of `docker build` invalidates the existing cache, and takes quite a long time to build.

i think moving it to just before `RUN ./install_ffmpeg.sh` line should do the trick. (instead of the very beginning, where we're just installing curl/apt/grpc etc)

Reply:
Got it, moved it just before `install_ffmpeg.sh`.